mon-pix/app/components/routes/campaigns/restricted/join-sco.hbs

```diff
@@ -36,7 +36,7 @@
   <div class="join-restricted-campaign__row">
     <label class="join-restricted-campaign__label">{{t 'pages.join.fields.birthdate.label'}}</label>
     <div class="join-restricted-campaign__birthdate">
-      <Input @id="dayOfBirth" @type="text" @value={{this.dayOfBirth}} placeholder={{t 'pages.join.fields.birthdate.day-format'}} class={{if this.validation.dayOfBirth "input--error"}} @focusOut={{ fn this.triggerInputDayValidation "dayOfBirth" this.dayOfBirth }} />
+      <Input @id="dayOfBirth" @type="text" @value={{this.dayOfBirth}} placeholder={{t 'pages.join.fields.birthdate.day-format'}} class={{if this.validation.dayOfBirth "input--error"}} @focusOut={{ fn this.triggerInputDayValidation "dayOfBirth" this.dayOfBirth }} autofocus={{this.isDisabled}} />
```

Comment:
Question : 
Pourquoi `autofocus={{this.isDisabled}}` ?

Reply:
Car quand on vient du GAR, les champs prénom et nom sont grisés.
Du coup c'est conditionné par le boolean this.disabled.

Reply:
Quelle est la différence entre `<Input {{autofocus}} />`et `<Input autofocus={{myBooleanVariable}} />` ?

Reply:
autofocus utilise un modifier dans notre application alors que autofocus = true est celui natif de html